rust/crd/src/lib.rs

```diff
@@ -272,16 +272,16 @@ impl MetaStoreConfig {
             warehouse_dir: None,
             resources: ResourcesFragment {
                 cpu: CpuLimitsFragment {
-                    min: Some(Quantity("200m".to_owned())),
-                    max: Some(Quantity("4".to_owned())),
+                    min: Some(Quantity("250m".to_owned())),
+                    max: Some(Quantity("1000m".to_owned())),
                 },
                 memory: MemoryLimitsFragment {
-                    limit: Some(Quantity("2Gi".to_owned())),
+                    limit: Some(Quantity("512Mi".to_owned())),
                     runtime_limits: NoRuntimeLimitsFragment {},
                 },
                 storage: MetastoreStorageConfigFragment {
                     data: PvcConfigFragment {
-                        capacity: Some(Quantity("2Gi".to_owned())),
+                        capacity: Some(Quantity("512Mi".to_owned())),
```

Comment:
Uhmm. We request a pvc without using (mounting) it :confused: 
I would remove the PVC entirely

Reply:
already done